src/ingredient/dto/create-ingredient.body.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsEnum, IsNotEmpty, IsNumber, IsString } from 'class-validator';
+import { IngredientUnit } from 'src/constants/enums/ingredient-unit.enum';
+
+export class CreateIngredientBody {
+  @IsString()
```

Comment:
let's follow convention of naming dto by file name, but camel case CreateIngredientBodyDto